tools/build-tools/README.md

```diff
@@ -13,30 +19,66 @@ In this directory:
 ```sh
 npm i
 npm run build
-npm link
+```
+
+## Running these tools (Command Line)
+
+This package produces several binaries, see `bin` in package.json.
+
+This repo is normally build using the version of this package referenced by the root package.json file.
+To run that version, just do one of:
+- `npm i` in the root, and use the npm scripts that call it (ex: `build:fast`)
+- globally install that specific version of `@fluidframework/build-tools` and call its binaries directly (ex `fluid-build`)
+
+There are several also ways to use the version of `@fluidframework/build-tools` from within the repo.
+Just built it (as in "Setup") then do one of:
+* Use (npm link)[https://docs.npmjs.com/cli/v8/commands/npm-link] with this package to override the version of it used in the root package (which is the `client` lerna package, but often used to build other as well). This will make scripts like `build:fast` use the linked version.
+* Use `--symlink:full`: Use some mechanism to run a working version of fluid-build with `--symlink:full`. This will setup the repo to use its included build tools for subsequent runs just like (npm link)[https://docs.npmjs.com/cli/v8/commands/npm-link], but also links everything else together.
```

Comment:
this links build-tools as well? i had no idea

Reply:
I though symlink:full would replace all references to packages which are in the repo to the actual version of the packages in the repo instead of ones from npm doing what lerna's symlinks does, cur across everything in the repo.

After running it and searching my filesystem for symlinks, I am convinced that is not what it does. It does something else: and I'm not sure what. With build-tools lacking a --version option, its hard to directly inspect what it does, but I'm pretty sure my proposed change is inaccurate.

Reply:
I have pushed corrections noting that this does not work.